### PR TITLE
removed deliver submodule and references to it

### PR DESCRIPTION
**Pull request by b97pla**
- Addresses #104
- Removed all traces of the deliver submodule which is now available as a plugin

### CODE REVIEW

**Comment by guillermo-carrasco** [comment]
uops, there is some conflict with my last PR, I guess that in the example config file? can you pull the last master branch version and solve the conflicts?

Tack tack!

**Comment by b97pla** [comment]
yep, I'll see if I can fix it

**Comment by b97pla** [comment]
@guillermo-carrasco that should do it

File: dependency_links.txt [hunk 1]
```diff
@@ -1,2 +1 @@
 https://github.com/SciLifeLab/statusdb/archive/master.zip#egg=statusdb
```
Comment (guillermo-carrasco) [inline dependency_links.txt]: I think that's not needed wither? or is it @senthil10 no I'm not sure... was it used in storage cleanup for UPPMAX maybe?

Comment (senthil10) [inline dependency_links.txt]: @guillermo-carrasco : Hej yes, it is still needed as I dint find the time to put `statusDB` in `Pypi` yet :sweat_smile:

Comment (guillermo-carrasco) [inline dependency_links.txt]: cool no problem ^^

**Comment by guillermo-carrasco** [comment]
I'll just wait for Travis to run the tests an merge then, thanks @b97pla !

**Comment by b97pla** [comment]
@guillermo-carrasco can this be merged?

**Comment by guillermo-carrasco** [comment]
wops, sure!

**Pull request merged**